README.md

```diff
@@ -314,27 +314,6 @@ StyleDictionary.buildAllPlatforms();
 
 For more information on creating your own transforms and formats, take a look at our [docs](https://styledictionary.com).
 
-## Version 4
```

Comment:
This statement still lives in the v4 -> statement.md but not really relevant anymore to be in the main README imo